src/core/operations_problem_results.jl

```diff
@@ -4,8 +4,7 @@ struct OperationsProblemResults <: IS.Results
     total_cost::Dict
     optimizer_log::Dict
     time_stamp::DataFrames.DataFrame
-    results_folder::Union{Nothing, String}
-    constraints_duals::Dict{Symbol, Any}
+    constraints_duals::Union{Nothing, Dict{Symbol, Any}}
```

Comment:
I reverted this change to be `Dict{Symbol, Any}` to keep consistency with the parameter_values